bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -101,66 +101,77 @@ <h1 class="large">{{_('Check Your Plugins')}}</h1>
       <p class="enable-js warning">{{_('Please turn on JavaScript to display this page correctly.')}}</p>
 
       <h2>{{_('Plugin Status')}}</h2>
-      <ul>
-        <li>{{_('Step 1: Click Update to update a plugin.')}}</li>
-        <li>{{_('Step 2: Complete all recommended updates before restarting your browser.')}}</li>
-      </ul>
 
-      <div class="plugincheck-loader"></div>
-
-      <div id="sec-plugin-vulnerable">
-        <table class="table status">
-          <caption>{{_('Potentially vulnerable plugins')}}</caption>
-          <thead>
-            <tr>
-              <th class="plugin-status-detail" scope="col">{{_('Plugin')}}</th>
-              <th class="plugin-status-status" scope="col">{{_('Status')}}</th>
-              <th class="plugin-status-action action" scope="col">{{_('Action')}}</th>
-            </tr>
-          </thead>
-          <tbody id="plugin-vulnerable"></tbody>
-        </table>
-      </div>
+      <div class="plugincheck-loader hidden"></div>
 
-      <div id="sec-plugin-outdated" class="shadow-separator">
-        <table class="table status">
-          <caption>{{_('Outdated Plugins')}}</caption>
-          <thead>
-            <tr>
-              <th class="plugin-status-detail" scope="col">{{_('Plugin')}}</th>
-              <th class="plugin-status-status" scope="col">{{_('Status')}}</th>
-              <th class="plugin-status-action action" scope="col">{{_('Action')}}</th>
-            </tr>
-          </thead>
-          <tbody id="plugin-outdated"></tbody>
-        </table>
+      {% if l10n_has_tag('no_plugins') %}
```

Comment:
I agree, the reason I left it with no else, was because currently there is no message if there are no plugins so, the current behavior does not change so, it felt like a safe fallback.